src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -38,7 +39,7 @@ class Auth(RESTController, ControllerAuthMixin):
     Provide authenticates and returns JWT token.
     """
     # pylint: disable=R0912
-    def create(self, username, password):
+    def create(self, username, password, ttl: Optional[int] = None):
```

Comment:
Done

Reply:
reverted that change, we cannot set it to default 8 hours here, because in that case every dashboard login would have a ttl of default 8 hrs even if we have set the jwt_token_ttl setting in the dashboard module. I have added the endpoint doc